### ISSUE TITLE
[BUG] Debian

### ISSUE THREAD

**Issue by sgnsys3** [konstruktoid/ansible-role-hardening#381]
**Describe the bug**
During running roles, this error is shown.

```
TASK [ansible-role-hardening : Set cron permissions] ***********************************************************************************************************************
ok: [localhost] => (item=/etc/cron.d)
ok: [localhost] => (item=/etc/cron.daily)
failed: [localhost] (item=/etc/cron.hourly) => {"ansible_loop_var": "item", "changed": false, "item": "/etc/cron.hourly", "msg": "file (/etc/cron.hourly) is absent, cannot continue", "path": "/etc/cron.hourly", "state": "absent"}
ok: [localhost] => (item=/etc/cron.weekly)
ok: [localhost] => (item=/etc/cron.monthly)
```

It looks like no directory /etc/cron.hourly on Debian-12

Is this normal behavior?

**To Reproduce**
Fresh install Debian 12 from cloud-init image

**System (lsb_release -a or similar):**
Distributor ID: Debian
Description:    Debian GNU/Linux 12 (bookworm)
Release:        12
Codename:       bookworm

**Comment by konstruktoid** [comment]
Hi @sgnsys3 and thanks for reporting this.
I'll have a look as soon as possible.

**Comment by konstruktoid** [comment]
Running the tests it seems that directory exists by default:
```sh
TASK [ansible-role-hardening : Set cron permissions] ***************************
changed: [bullseye] => (item=/etc/cron.d)
changed: [bookworm] => (item=/etc/cron.d)
changed: [testing] => (item=/etc/cron.d)
changed: [bullseye] => (item=/etc/cron.daily)
changed: [bookworm] => (item=/etc/cron.daily)
changed: [testing] => (item=/etc/cron.daily)
changed: [bullseye] => (item=/etc/cron.hourly)
changed: [bookworm] => (item=/etc/cron.hourly)
changed: [testing] => (item=/etc/cron.hourly)
changed: [bullseye] => (item=/etc/cron.weekly)
changed: [bookworm] => (item=/etc/cron.weekly)
changed: [bullseye] => (item=/etc/cron.monthly)
changed: [testing] => (item=/etc/cron.weekly)
changed: [bookworm] => (item=/etc/cron.monthly)
changed: [testing] => (item=/etc/cron.monthly)
```
but I'll write a PR that handles a missing ` cron` folder.

**Comment by konstruktoid** [comment]
could try with https://github.com/konstruktoid/ansible-role-hardening/pull/382?

**Comment by sgnsys3** [comment]
Hi, I have tried #382 The problem above was solved, and Found new error "/etc/crontab is absent. cannot continue"
The root cause is package cron was not installed on the OS.
In this case, Do we need to install cron prior to hardening or Check If cron is installed then hardening cron ?

**Comment by konstruktoid** [comment]
Ah, yes. `cron` is assumed to be installed. I'll rewrite the task so it only runs if cron is installed

**Comment by konstruktoid** [comment]
I've added the same check for `crontab`, https://github.com/konstruktoid/ansible-role-hardening/pull/382/files

**Comment by sgnsys3** [comment]
Hi I have tested this fix. It work well !
Thank you for your help !

**Comment by konstruktoid** [comment]
Closed by #382
